packages/gatsby/src/query/__tests__/file-parser.js

```diff
@@ -242,8 +258,23 @@ export default () => {
       Object.keys(MOCK_FILE_INFO),
       addError
     )
-    expect(results).toMatchSnapshot()
-    expect(reporter.warn).toMatchSnapshot()
+
+    // Check that invalid entries are not in the results and thus haven't been extracted
+    expect(results).toEqual(
+      expect.arrayContaining([
+        expect.not.objectContaining({ filePath: `no-query.js` }),
+        expect.not.objectContaining({ filePath: `other-graphql-tag.js` }),
+        expect.not.objectContaining({ filePath: `global-query.js` }),
+        expect.not.objectContaining({
+          filePath: `global-static-query-hooks.js`,
+        }),
+      ])
```

Comment:
These 2 are here because we panicked and did not get added to the list, correct?

Reply:
Correct!